gunrock/app/hits/hits_problem.cuh

```diff
@@ -31,21 +31,32 @@ cudaError_t UseParameters_problem(util::Parameters &parameters) {
   GUARD_CU(gunrock::app::UseParameters_problem(parameters));
 
   GUARD_CU(parameters.Use<int64_t>(
-      "max-iter",
+      "hits-max-iter",
       util::REQUIRED_ARGUMENT | util::MULTI_VALUE | util::OPTIONAL_PARAMETER,
-      100, "Number of HITS iterations.", __FILE__, __LINE__));
+      10000, "Number of HITS iterations.", __FILE__, __LINE__));
 
   GUARD_CU(parameters.Use<int64_t>(
-      "normalize-n",
+      "hits-norm",
+      util::REQUIRED_ARGUMENT | util::MULTI_VALUE | util::OPTIONAL_PARAMETER,
+      1, "Normalization method. 1 = normalize by the sum of absolute values (default), 2 = normalize by the square root of the sum of squares.", __FILE__, __LINE__));
+
+  GUARD_CU(parameters.Use<int64_t>(
+      "hits-normalize-n",
       util::REQUIRED_ARGUMENT | util::MULTI_VALUE | util::OPTIONAL_PARAMETER, 1,
-      "Normalize HITS scores every N iterations.", __FILE__, __LINE__));
+      "Normalize HITS scores and perform convergence checks every N iterations.", __FILE__, __LINE__));
```

Comment:
Maybe "and check for convergence" instead of  "convergence checks"?

Reply:
Fixed